### DISCUSSION TITLE
Unable to open .raw files

### DISCUSSION THREAD

**Discussion by ImagingOffice** [ImageMagick/ImageMagick#7183] [Digital Image Processing] [answered]
Hi,

I am working on a .raw image in de command line of a Windows 10 system. However, I already face troubles with loading the image. For example: 
`magick identify -verbose result.raw`
Gives the output: _identify: Unsupported file format or not RAW file 'result.raw' @ error/dng.c/ReadDNGImage/503_.

Anyone any suggestions?

Best

**Comment by fmw42** [comment]
Please post your raw image file.  From what camera was the image taken?  Do you have libraw installed.  What do you get from `magick -version`.

**Reply by ImagingOffice** [reply]
I will try to find an image that I am able to share as the image contains sensitive information. The image is originally a DICOM from an MRI and converted to .mhd and .raw during processing. 
I just installed libraw, but unsure if Magick is able to find it. However, the error occurs in line 503, whereas libraw is included in line 76. I would assume that an error would have occured if Magick is not able to find libraw?

This is the output from `magick -version`:
_Version: ImageMagick 7.1.1-29 Q16-HDRI x64 0deac72:20240225 https://imagemagick.org
Copyright: (C) 1999 ImageMagick Studio LLC
License: https://imagemagick.org/script/license.php
Features: Channel-masks(64-bit) Cipher DPC HDRI Modules OpenCL OpenMP(2.0)
Delegates (built-in): bzlib cairo flif freetype gslib heic jng jp2 jpeg jxl lcms lqr lzma openexr pangocairo png ps raqm raw rsvg tiff webp xml zip zlib
Compiler: Visual Studio 2022 (193833135)_

**Reply by ImagingOffice** [reply, marked as answer]
I think that my data is too complicated (i.e. not a 2D image) for this to work. Thank you anyway.

**Comment by snibgo** [comment]
ImageMagick can read DICOM (.DCM) files.

In ImageMagick, "raw" files usually mean files that come from a digital camera, and represent the data captured at the camera sensor without any processing. IM has tried to read your file as this type of raw file (ReadDNGImage), and rejected it.

Another meaning of "raw" is pixel data without any header. Each sample might be integer or floating-point, and one byte per sample or 2 bytes or 4 bytes, etc. There may be one sample per pixel (grayscale) or 3 samples per image (colour), with or without an alpha sample. IM can read and write this type of raw file. See https://www.imagemagick.org/script/formats.php#supported , formats GRAY, GRAYA, RGB and RGBA.
